app/controllers/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UsersController < ApplicationController
   before_action :set_user, only: %i[ show edit update destroy ]
+  before_action :set_logged_user, only: [:orders, :line_items]
```

Comment:
@singhal-ekansh why can't we use the existing `set_user` method.

Reply:
will change after reading routes